services/movie_session.py

```diff
@@ -0,0 +1,44 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
```

Comment:
not needed to `return` anything here

Reply:
i think it was a good practice because of how method create work so i wrote returns 
<img width="479" alt="image" src="https://user-images.githubusercontent.com/123869844/225075360-4c7c2187-c336-41e3-ba18-340e98fce180.png">

Reply:
am i wrong?

Reply:
Of course ORM's method `create` returns an object that was created, because you sometimes need something like that: `var = Model.objects.create(...)`. But you are not implementing `create` method for framework's ORM :)
All you need is to have method that will create MoviSession record in db, that' it

Reply:
got u bro <3

Reply:
thx a lot for explain